Applications/DataExplorer/DataView/MeshElementRemovalDialog.cpp

```diff
@@ -132,19 +139,18 @@ void MeshElementRemovalDialog::on_boundingBoxCheckBox_toggled(bool is_checked)
 	if (is_checked && (_currentIndex != _aabbIndex))
 	{
 		_aabbIndex = _currentIndex;
-		const std::vector<MeshLib::Node*> nodes (_project.getMesh(this->meshNameComboBox->currentText().toStdString())->getNodes());
+		std::vector<MeshLib::Node*> const& nodes (_project.getMesh(this->meshNameComboBox->currentText().toStdString())->getNodes());
 		GeoLib::AABB<MeshLib::Node> aabb(nodes.begin(), nodes.end());
-		const MeshLib::Node minAABB = aabb.getMinPoint();
-		const MeshLib::Node maxAABB = aabb.getMaxPoint();
+		MeshLib::Node const minAABB = aabb.getMinPoint();
+		MeshLib::Node const maxAABB = aabb.getMaxPoint();
 		this->xMinEdit->setText(QString::number(minAABB[0], 'f'));
 		this->xMaxEdit->setText(QString::number(maxAABB[0], 'f'));
 		this->yMinEdit->setText(QString::number(minAABB[1], 'f'));
 		this->yMaxEdit->setText(QString::number(maxAABB[1], 'f'));
 		this->zMinEdit->setText(QString::number(minAABB[2], 'f'));
 		this->zMaxEdit->setText(QString::number(maxAABB[2], 'f'));
-
+		std::fill(aabb_edits.begin(), aabb_edits.end(), false);
```

Comment:
Just for information: the line can be a little shorter with `aabb_edits.fill(false);`, but is ok the way it is.